src/OrdersList/OrdersDriver.cpp

```diff
@@ -3,7 +3,6 @@
 #include "../Player/Player.h"
 #include "../Map/Map.h"
 
-
 void testOrdersLists() {
     cout << "\n" << "************************************" << "\n"
```

Comment:
instead of 
```js 
    cout << "\n" << "************************************" << "\n"  << "Time to test Orders and OrdersList!! \n"
```

this can be also as one string insted of creating a few streams:
```js
    cout << "\n************************************"\n Time to test Orders and OrdersList!! \n"

```

Reply:
I did this for visuals, it's not easier to read?